accounts/abi/bind/v2/backend.go

```diff
@@ -0,0 +1,20 @@
+package v2
+
+import (
+	"context"
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"math/big"
+)
+
+type V2Backend interface {
```

Comment:
I think we should rename this to `BackendV2`, we've been using V2 as a post-fix almost everywhere and should stay consistent with it

Reply:
SGTM